README.md

```diff
@@ -1 +1,69 @@
-# sawtooth-mktplace
\ No newline at end of file
+# Sawtooth Marketplace
```

Comment:
In a future PR, most of this should move to sphinx-doc so it can be published.

Reply:
I don't see sphinx and published docs as a necessity for these self contained example apps that are intended to be cloned and modified.